packages/docs/verite/patterns/smart-contract-verite.md

```diff
@@ -88,7 +90,7 @@ The registry provides external-scoped accessor methods to obtain VerifierInfo fo
 
 ## Verifier Submission Pattern
 
-Either a verifier or a subject may submit a Verification Result for validation. Which pattern is appropriate depends upon the use case in question.
+Either a verifier or a subject may submit a Verification Result to a dapp directly for validation. Which pattern is appropriate depends upon the use case in question.
```

Comment:
In theory a verifier or subject _could_ send a VR directly to a smart contract, which performs the validation, although -- especially for subject-submission -- this would be pretty rare.

That's context for a dumb question: does colloquial use of "dapp" also allow the interpretation a no-front-end / all-back-end (smart contract) situation?

Reply:
edited for clarity